_posts/2018-02-20--inception-pipelines-pt1.markdown

```diff
@@ -0,0 +1,99 @@
+---
+layout: post
+title:  "Seeds of Inception - Part 1: Seeding your Account with an Inception Pipeline"
+date:   2018-02-20
+categories: aws continuous deployment
+author: Pete Yandell
+image: img/inception-pipelines/inception-pipeline-cover.png
+---
+
+## What's the problem
+
+As we all know, we are supposed to automate everything, every must be 'as code' and there are no manual steps to doing anything. However when pressed with project deadlines, production fires and the rare moments when we get to work on something new, we just don't get around to it.
+
+While working on a recent project, building out a continuous deployment pipeline, I got to thinking about the number of upcoming future projects. Each project would need at least one pipeline per application, and a dedicated one for the AWS Account. Rolling a unique snowflake pipeline everytime just seemed evil and wasteful and wrong and totally against everything we stand for a [Mechanical Rock](https://www.mechanicalrock.io)!
+
+So welcome to the Inception Pipeline; a [CloudFormation](https://aws.amazon.com/cloudformation/) template that plants itself inside an AWS Account and then self manages and self updates itself using nothing more than off-the-shelf AWS services.
+
+## What technologies are we going to use
```

Comment:
Title case for headings?

Sentence case looks a bit weird

Reply:
Fixed. I was on the fence for this one.